angular.json

```diff
@@ -100,10 +107,7 @@
         "lint": {
           "builder": "@angular-devkit/build-angular:tslint",
```

Comment:
This is failing. We should update the builder to use `"@angular-eslint/builder:lint"`